.github/workflows/test.yml

```diff
@@ -0,0 +1,42 @@
+name: tests
+
+on:
+  pull_request:
+    types: ['opened', 'synchronize']
+    paths-ignore:
+      - 'README.md'
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - 'README.md'
+jobs:
+  unit:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: true
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      -
+        name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.15
+      - 
```

Comment:
We should also run check format and linting as part of the CI too. Although this can wait for PR of its own because enabling that will absolutely break the CI and we will never have a green status until those are fixed.

Reply:
Yes sure. I created an issue for this: https://github.com/git-chglog/git-chglog/issues/89

Reply:
That's awesome, thanks @mavogel!